styles/styles.css

```
@@ -0,0 +1,567 @@
+}
+
+
+.header-content {
+    flex: 0 1 42rem;
+    text-align: center;
+    margin: 2rem 1rem;
+}
+
+.squiggle-container {
+    flex: 0 1 4rem;
+    /*shorthand for background*/
+    background: url(../assets/squiggle_line_stripe.svg) bottom / contain no-repeat;
+}
+
+.header-button {
+    background-color: var(--button--color--);
+    padding: 1.5rem 2.5rem;
+    margin: 0.75rem;
+    border: none 50px;
+    border-radius: 50px;
+    color: white;
+    text-transform: uppercase;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.bulletpoint-container {
+
+    flex-direction: column;
+    flex: 0 1 46rem;
+    justify-content: space-between;
+    text-align: center;
+}
+
+.bulletpoints {
+    color: var(--emphasized-text-color--);
+    margin: 8rem 0 3.5rem 0;
+}
+
+/*styles the individual bulletpoints*/
+.bulletpoints h3 {
+    margin: var(--default-margin--);
+}
+
+.contact-us-link {
+    margin-bottom: 8rem;
+}
+
+/* --------------- 4.3  sections ------------------- */
+
+section {
+    position: relative;
+    background-color: var(--background-uneven-main-sections--);
+}
+
+/*responsible for alternating background colours*/
+section:nth-child(even) {
+    background-color: var(--background-even-main-sections--);
+}
+
+/*creates slanted lines for the sections*/
+section::before {
+    position: absolute;
+    content: "";
+    display: block;
+    clip-path: polygon(0 65%, 0 100%, 100% 100%);
+    width: 100%;
+    height: 10rem;
+    background-color: var(--background-uneven-main-sections--);
+    z-index: 1;
+    top: -9.9rem;
+}
+
+/*alternates triangle colours*/
+section:nth-child(even)::before {
+    background-color: var(--background-even-main-sections--);
+}
+
+/*adds a white triangle to the very last section*/
+section:last-of-type::after {
+    position: absolute;
+    content: "";
+    display: block;
+    clip-path: polygon(0 65%, 0 100%, 100% 100%);
+    transform: rotate(180deg);
+    width: 100%;
+    height: 10rem;;
+    z-index: 40;
+    bottom: -9.9rem;
+}
+
+/*checks wether the last section is odd or even to apply the proper bg colour*/
+section:last-of-type:nth-child(odd)::after {
+    background-color: var(--background-uneven-main-sections--);
+}
+
+/*checks wether the last section is odd or even to apply the proper bg colour*/
+section:last-of-type:nth-child(even)::after {
+    background-color: var(--background-even-main-sections--);
+}
+
+/*creates the background text*/
+.section-a .inner-container::after {
+    content: "hello";
+    color: var(--background-word-color-one--);
+    z-index: -1;
+    position: absolute;
+    top: 24%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    font-size: 12rem;
+    font-weight: 500;
+}
+
+.section-a h4 {
+    margin: 0 1rem 1.5rem 1rem;
+}
+
+.section-b .inner-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    flex: 0 0 30rem;
+    margin: 10rem 0 10rem 0;
```

Dit zou je korter kunnen schrijven
```suggestion
    margin: 10rem 0;
```